### DISCUSSION TITLE
blog/bai-viet-dau-tien-tren-blog-moi

### DISCUSSION THREAD

**Discussion by giscus** [kmacoders/kmacoders.github.io#4] [General]
# blog/bai-viet-dau-tien-tren-blog-moi

Bài viết này thực ra được viết từ 10/08/2019. Đây cũng là bài viết đầu tiên tại blog kmacoders.github.io ngày ấy, thời điểm mà mình biết những dòng mà mình đang gõ ra đã có những người đầu tiên đọc.

http://localhost:3000/blog/bai-viet-dau-tien-tren-blog-moi

**Comment by kmacoders** [comment]
Test first comment !!!
